flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/JobObserver.java

```diff
@@ -87,8 +87,8 @@ private void updateJobStatus(JobStatus status, List<JobStatusMessage> clusterJob
         status.setState(newJob.getJobState().name());
         status.setJobName(newJob.getJobName());
         status.setJobId(newJob.getJobId().toHexString());
-        // track the start time, changing timestamp would cause busy reconciliation
-        status.setUpdateTime(String.valueOf(newJob.getStartTime()));
+        status.setStartTime(String.valueOf(newJob.getStartTime()));
+        status.setUpdateTime(String.valueOf(System.currentTimeMillis()));
```

Comment:
Have you confirmed that this does not cause a busy reconcile loop? I actually think we should just rename the existing field.

Reply:
The current javaoperatorsdk behavior is a little strange. When `generationAwareEventProcessing` is configured to `false`, not `true`, then the reconciliation will not be triggered if `.spec` is not changed.

[1]. https://javaoperatorsdk.io/docs/features

Reply:
@wangyang0918 exactly, we only want to trigger new reconcile automatically when the Spec itself is changed by the user. This does not affect reconciliations that we schedule ourselves but avoids immediate reconciliation triggers when the status changes.

Reply:
GenerationAware here means that it is aware that we should only reconcile if the generation (spec) changes. So it might be a bit confusing but it seems to work :)

Reply:
@gyfora So do you mean to need to set `generationAwareEventProcessing` to `false` to enable the filtering? I am afraid the default value is `true`.

Reply:
we want it to be aware of the generation so we want it to be `true` which is the default. True enables filtering, false triggers reconcile for every spec/status change

Reply:
The description in the java doc[1] is not consistent with the documentation.

[1]. https://github.com/java-operator-sdk/java-operator-sdk/blob/main/operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/reconciler/ControllerConfiguration.java#L27

![image](https://user-images.githubusercontent.com/15904523/158537063-3f7c75fd-6832-4d7f-8ffb-7de4e800c445.png)

Reply:
Yea it should say, `To turn off this feature...`

Reply:
I have to verify whether it really works.

Reply:
FYI: it is indeed a typo in the documentation. They have fixed this.

I have verified the status changes will not cause busy reconciliation.